### PR TITLE
BaSP-MR-167: [Admin] Reports

### PR DESCRIPTION
**Pull request by marcohzo**
- Create a path for reports inside the admin folder
- 3 navigation buttons created
- The activities screen show a graph with each activity loaded in the database with its respective quantity.
- The memberships screen display a graph with each memberships loaded into the database in each month.
- The members screen display a graph with each member loaded into the database in each month.

### CODE REVIEW

File: src/Components/Admin/Reports/Activities/index.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    if (Object.keys(activityCount).length === 0) {
+      activityCount['No activity'] = 1;
+    }
+  }
+
+  const uniqueActivities = subscriptions
+    ? Array.from(new Set(subscriptions.map((item) => item.classes?.activity?.name)))
+    : [];
+
+  console.log(uniqueActivities);
```
Comment (AleMac17) [inline src/Components/Admin/Reports/Activities/index.jsx]: We should not leave console logs in the code

File: src/Components/Admin/Reports/Activities/index.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  const activityCount = {};
+
+  if (subscriptions) {
+    subscriptions.forEach((item) => {
+      const activityName = item.classes?.activity?.name || 'No activity';
+      activityCount[activityName] = (activityCount[activityName] || 0) + 1;
+    });
+
+    // Verificar si no se encontraron actividades
```
Comment (AleMac17) [inline src/Components/Admin/Reports/Activities/index.jsx]: We should not leave comments in the code

**Review by AleMac17 — CHANGES_REQUESTED** [review]

**Review by AleMac17 — APPROVED** [review]

**Comment by fernandofernandezuy** [comment]
Looks great, can you put the activities report to the left? and give styles to the container?

**Review by fernandofernandezuy — APPROVED** [review]
Looks good!, great job

**Review by ale-schmidt — APPROVED** [review]
Looks and works great!

**Review by Gonzarey96 — APPROVED** [review]

**Comment by fernandofernandezuy** [comment]
[[Admin: Reports] Create Admin > Reports screen](https://trello.com/c/CXD0MG4A/167-admin-reports-create-admin-reports-screen)

**Comment by nazarenapantanetti** [comment]
Tested in master, changes requested! Follow the link to see the full report: https://trello.com/c/ZWzxF7pp/179-reports-bug-graph-is-not-clear

**Pull request merged**